File: backend/tests/fixtures/parsers/lua/hello.lua
local m = require("utils")

Greeter = {}

function Greeter:greet(name)
    local formatted = self:format_name(name)
    print(formatted)
end

function Greeter:format_name(name)
    return m.upper(name)
end

local function standalone_helper()
    Greeter:greet("world")
end

File: backend/tests/fixtures/parsers/zig/hello.zig
const std = @import("std");

const Greeter = struct {
    name: []const u8,
};

fn greet(name: []const u8) []const u8 {
    return formatName(name);
}

fn formatName(n: []const u8) []const u8 {
    return n;
}

fn standalone() void {
    _ = greet("world");
}
